public/js/icinga/events.js

```diff
@@ -457,6 +457,12 @@
                 return true;
             }
 
+            // Check for ctrl or cmd click to open new tab
+            if (event.ctrlKey || event.metaKey) {
```

Comment:
What's the `metaKey`? Is this an apple specific thing?

Reply:
Form the [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey)
On Macintosh keyboards, this is the ⌘ Command key. On Windows keyboards, this is the Windows key (⊞ Windows).